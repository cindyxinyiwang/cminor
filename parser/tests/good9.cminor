// corner case

printarray: function void ( a: array [] integer, size: integer ) = {
	print;
	return;
}