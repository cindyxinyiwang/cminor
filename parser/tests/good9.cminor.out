printarray: function void ( a: array [] integer, size: integer ) = {
	print;
	return;
} : function void(a: array [] integer, size: integer ) = {
	print;
	return;
}: array [] integer,size: integer ) = {
	print;
	return;
}: integer)= {
	print ;
	return ;
}
